common/static/common/js/spec/components/feedback_spec.js

```diff
@@ -114,6 +115,72 @@
       });
     });
     describe("PromptView", function() {
+      beforeEach(function() {
+        this.options = {
+          title: "Confirming Something",
+          message: "Are you sure you want to do this?",
+          actions: {
+            primary: {
+              text: "Yes, I'm sure.",
+              "class": "confirm-button",
+            },
+            secondary: {
+              text: "Cancel",
+              "class": "cancel-button",
+            }
+          }
+        }
+        this.inFocusSpy = spyOn(PromptView.Confirmation.prototype, 'inFocus').andCallThrough();
+        return this.outFocusSpy = spyOn(PromptView.Confirmation.prototype, 'outFocus').andCallThrough();
+      });
+      it("is focused on show", function() {
+        var view;
+        view = new PromptView.Confirmation(this.options).show();
+        expect(this.inFocusSpy).toHaveBeenCalled();
+        return waitsFor(
```

Comment:
Can you refactor this into a helper method for verifying that an element has the focus. Look at view_helpers.js for similar helper functions. In fact, it should be two helpers (or one parameterized one) to test having and not having focus.

https://github.com/edx/edx-platform/blob/master/common/static/common/js/spec_helpers/view_helpers.js